satpy/readers/seviri_l2_bufr.py

```diff
@@ -171,30 +171,29 @@ def get_array(self, key):
         return arr
 
     def get_dataset(self, dataset_id, dataset_info):
-        """Get dataset using the BUFR key in dataset_info."""
+        """Get dataset using the BUFR key in dataset_info. The create dataset with or without an AreaDefinition."""
```

Comment:
`The create dataset with or without an AreaDefinition.`

What did you mean here?

Reply:
Good question, probably it should have been `Then ...`. I have revised the docstring in to make it more clear.